docs/storefront/cart-checkout/guide/graphql-storefront.mdx

```diff
@@ -575,8 +575,9 @@ If using the getCart query with customer impersonation tokens, you must include
 </Tabs>
 
 ### Create cart metafield
+
 <Callout type="info">
-The current platform limit is 250 metafields for a single cart. 
+The current platform limit is 250 metafields for a single cart. When you create metafields using the `createCartMetafield` mutation, the default namespace is `bc_storefront`.
```

Comment:
@harnerdesigns is this description what you meant by "this way"?

Reply:
@slsriehl, yup! That explains it better. 👍🏽